src/ios/CDVUIInAppBrowser.m

```diff
@@ -348,9 +358,6 @@ - (void)loadAfterBeforeload:(CDVInvokedUrlCommand*)command
 
 - (void)injectDeferredObject:(NSString*)source withWrapper:(NSString*)jsWrapper
 {
-    // Ensure an iframe bridge is created to communicate with the CDVUIInAppBrowserViewController
-    [self.inAppBrowserViewController.webView stringByEvaluatingJavaScriptFromString:@"(function(d){_cdvIframeBridge=d.getElementById('_cdvIframeBridge');if(!_cdvIframeBridge) {var e = _cdvIframeBridge = d.createElement('iframe');e.id='_cdvIframeBridge'; e.style.display='none';d.body.appendChild(e);}})(document)"];
```

Comment:
Is `createIframeBridge` equal to this?
Is it safe to move this to where `createIframeBridge` is called now?

Reply:
> Is `createIframeBridge` equal to this?
Yes, this code has been factorised out into `createIframeBridge`

> Is it safe to move this to where `createIframeBridge` is called now?
In this original implementation, the iframe bridge is created "on demand", the first time that an injection method (such as `injectScriptCode`) is called. Subsequent calls find the iframe bridge already exists, so it's only created once.

In the new implementation, `createIframeBridge` is called after every successful page load (`webViewDidFinishLoad` to ensure the bridge always exists, regardless of whether an injection method has been called. This enables pages contained within the IAB Webview to make use to the bridge in order to send `message` events back to the app Webview, without relying on an injection method having been previously called in order to create the bridge.

Therefore, I think it's safe enough to do this.